01week/bio/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Samantha's Bio Page</title>
+    <link href="style.css" rel="stylesheet" type="text/css" />
+</head>
+<body>
+
+<div id="wrapper">
+
+	<div id="header">
+      <img src="https://unsplash.com/?photo=21ae7tGpJtE"
+		<div id="nav"><a href="index.html">Home</a><br></br><a href="#">About Me</a><br></br>
+      <a href="#">Contact Me</a<br></br>
```

Comment:
You need an > at the end of your closing anchor tag.  Also, line breaks don't need a closing tag.